docs/platforms/flutter/enriching-events/viewhierarchy/index.mdx

```diff
@@ -21,6 +21,32 @@ View hierarchy debugging is an opt-in feature. You can enable it as shown below:
 
 <PlatformContent includePath="enriching-events/attach-viewhierarchy" />
 
+### Customize View Hierarchy Capturing
+
+<Note>
+
+Requires SDK version `8.12.0` or higher.
+
+</Note>
+
+Because capturing view hierarchy can be a resource-intensive operation on Flutter, it's limited to one view hierarchy every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for view hierarchies in the `SentryFlutterOptions`.
+
+The `BeforeCaptureCallback` also allows you to customize behavior based on event data so you can decide when to capture view hierarchy and when not to. For example, you can decide to only capture view hierarchy for fatal events:
+
+```java
+await SentryFlutter.init((options) {
+  options.attachViewHierarchy = true;
+  options.beforeCaptureViewHierarchy = (event, hint, debounce) async {
+    // If debounce is active, skip capturing
+    if (debounce) {
+      return false;
+    }
```

Comment:
iirc that means that users can also skip debouncing if they return true anyway, right?

Reply:
Correct, it's the same API that we have on Android. it's also mentioned in the docs AFAIK.